power_grid_model_c/power_grid_model/include/power_grid_model/main_core/update.hpp

```diff
@@ -130,4 +135,194 @@ inline void update_inverse(MainModelState<ComponentContainer> const& state, Forw
                                      get_component_sequence<Component>(state, begin, end));
 }
 
+namespace update_independence {
+namespace detail {
```

Comment:
Maybe (or maybe not) a better name for this namespace is needed. Suggestions are welcome.

Reply:
`detail` is fine I think